tests/test_abstract_ccxt_pair_converter.py

```diff
@@ -74,10 +81,11 @@ def unoptimized_graph(self, vertex_list: Dict[str, Vertex[str]]) -> MappedGraph[
 
     @pytest.fixture
     def historical_bars(self) -> Dict[str, HistoricalBar]:
+        now_time = datetime.now(timezone.utc)
         return {
             MARKET_START: HistoricalBar(
                 duration=timedelta(weeks=1),
-                timestamp=datetime(2023, 1, 1),
+                timestamp=now_time,
```

Comment:
Since this is a test wouldn't it be better to have fixed data (like 2023/1/1) rather than datetime.now()?

Reply:
I don't think I can since the guard clause checks for stale data. It needs to be a timestamp within the last week. See other comment.